### ISSUE TITLE
Figure out best way to handle RDF

### ISSUE THREAD

**Issue by jonc125** [ModellingWebLab/WebLab#13]
For metadata, it'd be nice to handle this properly as RDF and support RDF querying methods, rather than hacking in support as with the prototype (model metadata just as RDF/XML within the files; protocol metadata as URIs in the DB).

Possibly we need to run a proper triplestore as well as Django's postgres DB?

RDFlib may be the best library for accessing & processing.

Need to investigate further!

Some initial links:
* http://www.dataversity.net/triplestores-101-storing-data-efficient-inferencing/
* https://lawlesst.github.io/notebook/rdflib-stardog.html

**Comment by MichaelClerx** [comment]
I like the idea of being able to relate any two things that have a url!

**Comment by MichaelClerx** [comment]
Are we still hoping to use the same standard as eventually adopted by COMBINE?

**Comment by jonc125** [comment]
Yes, at least for how annotations are associated with models in files. Whether this extends to use of particular ontologies...
